src/ray/gcs/gcs_client/test/gcs_client_test.cc

```diff
@@ -115,6 +115,15 @@ class GcsClientTest : public ::testing::TestWithParam<bool> {
     rpc::ResetServerCallExecutor();
   }
 
+  void ReconnectClient() {
+    ClusterID cluster_id = gcs_server_->rpc_server_.GetClusterId();
```

Comment:
Can we also provide a GetClusterId method so we don't need friend class in gcs_server as well?